public/paths/products/products.yml

```diff
@@ -0,0 +1,98 @@
+get:
+  tags:
+    - Products
+  summary: List products
+  description: Receive list of products saved in the database.
+  parameters:
+  - in: query
+    name: _sort
+    required: false
+    example: 'id'
+    schema:
+      type: string
+      enum:
+      - id
+      - firstName
+      - lastName
+      - swedishSocialSecurityNumber
```

Comment:
This looks wrong. That should be the list of fields that can be sorted on. I thik it is only riskScore and createdAt